### PR TITLE
Change disclaimer wording.

### PR DESCRIPTION
**Pull request by dzimine**

### CODE REVIEW

**Comment by manasdk** [comment]
+1

File: tools/st2_deploy.sh [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
 read -r -d '' WARNING_MSG << EOM
 ######################################################################
-######                       WARNING                           #######
+######                       DISCLAIMER                        #######
 ######################################################################
 
-This scripts allows you to evaluate StackStorm on a single server and
-is not intended to be used for production deployments.
+This scripts installs StackStorm on a single server. Check the docs
+for multi-server deployemnt, hardening security, and other aspects of
```
Comment (Kami) [inline tools/st2_deploy.sh]: deployemnt -> deployment

Comment (manasdk) [inline tools/st2_deploy.sh]: Done :P

**Comment by lakshmi-kannan** [comment]
@dzimine It turned out to be triple work. Thanks for this :P.

**Comment by manasdk** [comment]
lol. I think this is fine now. We can change when someone else complains about the disclaimer.

**Pull request merged**